src/sass/_theme_font_awesome_compatibility.sass

```diff
@@ -1,24 +0,0 @@
-.icon
-  @extend .fa
```

Comment:
I maybe have slight concern here about removing the class, in case anyone downstream is using the `.icon` classes. This might be another backwards incompatible change